### PR TITLE
Fix formatting in README

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,11 +9,14 @@ Para pré-visualizar localmente:
...
+    ```bash
     virtualenv venv
     source venv/bin/activate
     pip install -r requirements.txt
-
+    ```
 
 2) Rode o servidor local do mkdocs:
 
+    ```
```
Comment (rougeth) [inline README.md]: @nicoddemus missing `bash`? :P

**Comment by nicoddemus** [comment]
I was about to fix the missing bash in the second block comment; I missed it initially because there was no visual difference because `mkdocs serve` is not bash syntax. 😋 

But that's OK, as I said there's no visual difference.

**Comment by rougeth** [comment]
Oh, you got my comment! hahaha I removed it because I didn't want to be annoying without a reason :stuck_out_tongue_winking_eye:

**Comment by nicoddemus** [comment]
Nah no worries about requesting changes, it is all part of the process. I didn't respond right away because I was cloning my fork to amend my commit. 😉 

Well all good!

**Comment by eliasdorneles** [comment]
Como o conteúdo é em português, a comunidade é brasileira, diria que é bom manter as discussões em português, pra deixar amigável para todos os contribuidores em potencial. :)

**Comment by nicoddemus** [comment]
@eliasdorneles bom ponto! Estou acostumado a só usar inglês no GH então meu comentário foi automático.

**Pull request merged**